Week2/6.Arrays.js

```diff
@@ -0,0 +1,23 @@
+// 6.1
+let animals = [] ;
+
+// 6.2
+console.log("the value of my array : Samoyed dog, British Shorthair, Horse") ;
```

Comment:
Is the value of the array that value right now? What is the value of `animals` right now? if I `console.log` it?